var/spack/repos/builtin/packages/melissa/package.py

```diff
@@ -18,8 +18,8 @@ class Melissa(CMakePackage):
     # attention: Git**Hub**.com accounts
     maintainers("christoph-conrads", "raffino")
 
-    version("master", branch="master")
-    version("develop", branch="develop")
+    version("master", branch="master", deprecated=True)
+    version("develop", branch="develop", deprecated=True)
     version("0.7.1", sha256="c30584f15fecf6297712a88e4d28851bfd992f31209fd7bb8af2feebe73d539d")
```

Comment:
You'll also want to deprecate this one if you're trying to remove this recipe entirely in a future release